test/test_EKF_initialization.cpp

```diff
@@ -108,6 +108,25 @@ class EkfInitializationTest : public ::testing::Test {
 		EXPECT_TRUE(vel_var(1) > vel_variance_limit) << "vel_var(1)" << vel_var(1);
 		EXPECT_TRUE(vel_var(2) > vel_variance_limit) << "vel_var(2)" << vel_var(2);
 	}
+
+	void learningCorrectAccelBias()
+	{
+		Dcmf R_to_earth = Dcmf(_ekf->getQuaternion());
+		Vector3f dvel_bias_var = _ekf_wrapper.getDVelBiasVariance();
+		Vector3f accel_bias = _ekf->getAccelBias();
+
+		for (int i = 0; i < 3; i++){
+			if (fabsf(R_to_earth(2, i)) > 0.8f) {
+				// Highly observable, the variance decreases
+				EXPECT_LT(dvel_bias_var(i), 2.0e-6f) << "axis " << i;
+
+			} else {
+				// Poorly observable, the variance is set to 0
+				EXPECT_FLOAT_EQ(dvel_bias_var(i), 0.f) << "axis" << i;
+				EXPECT_FLOAT_EQ(accel_bias(i), 0.f) << "axis" << i;
```

Comment:
Is the accel bias set to zero for the unobservable states? Or does it stay constant?

Reply:
The learned stays constant, but since it's just after initialization, we can assume that it is zero. The variance however is forced to zero and saved in a variable for re-use when observable again.